testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetHostedIT.java

```diff
@@ -175,4 +176,23 @@ public void visualStudioSearchForUpdates() throws Exception {
     assertThat("entry count", entries.size(), is(1));
     assertThat("entry ID", entries.get(0).get("ID"), is("SONATYPE.TEST"));
   }
+
+  /**
+   * Ensure that an ODATA query for an uploaded package's metadata
+   * returns an entry that contains a valid content link to the package.
+   */
+  @Test
+  public void link_to_package_is_valid_in_package_metatdata_returned_from_url_encoded_ODATA_query() throws Exception {
```

Comment:
Would prefer this to be a shorter camelcase name and depend on the jdoc for detail. Can't say this style is actually against our coding guidelines, but it would certainly be a departure from the norm.